model/model.js

```diff
@@ -88,4 +88,31 @@ exports.selectCommentsByArticleId = (articleId) => {
     }
     return comments.rows
   })
+}
+
+exports.insertCommentByArticleId = (articleId, newComment) => {
+  const articleIdNum = Number(articleId)
```

Comment:
You should have psql error handlers checking this. No need for custom rejects as the database and seed will tell you what it requires.